crates/router/src/connector/stripe/transformers.rs

```diff
@@ -247,11 +276,42 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for PaymentIntentRequest {
             None => StripeShippingAddress::default(),
         };
 
+        let billing_address = match item.address.billing.clone() {
+            Some(mut billing) => StripeBillingAddress {
+                email: item.request.email.clone(),
+                country: billing
+                    .address
+                    .as_ref()
+                    .and_then(|address| address.country.clone()),
+                name: billing.address.as_mut().and_then(|a| {
+                    a.first_name.as_ref().map(|first_name| {
+                        format!(
+                            "{} {}",
+                            first_name.clone().expose(),
+                            a.last_name.clone().expose_option().unwrap_or_default()
+                        )
+                    })
+                }),
+                line1: billing
+                    .address
+                    .as_ref()
+                    .and_then(|address| address.line1.clone()),
+            },
+            None => StripeBillingAddress::default(),
+        };
+
+        crate::logger::debug!(?billing_address);
```

Comment:
Is the address being logged intentional?

Reply:
Think of debug logs from production deployment perspective. How will you use this to debug a production issue?